c2corg_ui/templates/outing/helpers/detailed_outings_attributes.html

```diff
@@ -0,0 +1,199 @@
+<%namespace file="../../helpers/view.html" import="show_areas, show_attr, show_maps, get_document_up_down, get_document_min_max"/>
+
+
+## GENERAL
+<%def name="get_outing_general(outing)">\
+<div class="name-icon-value general" ng-click="detailsCtrl.openTab($event)">
+  <h4><span translate>General</span><span class="glyphicon glyphicon-menu-right"></span></h4>
+  <div class="name-icon">
+    <span class="glyphicon glyphicon-info-sign"></span>
+    <p translate>General</p>
+  </div>
+  <span class="detail-text accordion">
+
+    <article ng-init="activities = ${outing['activities']}" class="value activities">
+      <b translate class="value-title">activities</b><br>
+      <b class="green-text" ng-repeat="type in activities">{{type | translate}}{{$last ? '' : ', '}}</b>
+    </article>
+    
+    <p class="value-title" ><span translate>outing of</span> <span> {{ '${outing['date_start']}' | amDateFormat:'Do MMMM YYYY'}}</span></p>
+    % if outing.get('awesomeness'):
+    <p><span class="value-title" translate>awesomeness</span>: <span class="value">${outing['awesomeness']}</span></p>
+    % endif
+    
+    % if outing.get('duration'):
+    <p><span class="value-title" translate>duration</span>: <span class="value">${outing['duration']}</span></p>
+    % endif
+    
+    % if outing.get('duration_difficulties'):
+    <p><span class="value-title" translate>duration_difficulties</span>: <span class="value" translate>${outing['duration_difficulties']}</span></p>
+    % endif
+    
+    % if outing.get('frequentation'):
+    <p><span class="value-title" translate>frequentation</span>: <span class="value" translate>${outing['frequentation']}</span></p>
+    % endif
+    
+    %if outing.get('condition_rating'):
+    <p><span class="value-title" translate>condition_rating</span>: <span class="value" translate>${outing['condition_rating']}</span></p>
+    % endif
+    
+    % if outing.get('partial_trip'):
+    <p><span class="value-title" translate>partial_trip</span>: <span class="value" translate>${outing['partial_trip']}</span></p>
+    % endif
+  </span>
+</div>
+</%def>
+
+## LOCATION
+<%def name="get_outing_location(waypoint, geometry4326)">\
+<div class="name-icon-value " ng-click="detailsCtrl.openTab($event)">
+  <h4><span translate>Location</span><span class="glyphicon glyphicon-menu-right"></span></h4>
+  <div class="name-icon">
+    <span class="glyphicon glyphicon-map-marker"></span>
+    <p translate>Location</p>
+  </div>
+  <span class="detail-text accordion" ng-init="point = ${geometry4326['geom']}">
+  x: {{point.coordinates[0] | number: 0}}
+  y:  {{point.coordinates[1] | number: 0}}
```

Comment:
We don't need this for outings: the default point geometry is only used to recenter the map or make geographic filtering. But having the coordinates of a point is not relevant for outings: they are not points.